chain/default_syncer.go

```diff
@@ -326,7 +326,7 @@ func (syncer *DefaultSyncer) widen(ctx context.Context, ts types.TipSet) (types.
 // attempt to validate and caches invalid blocks it has encountered to
 // help prevent DOS.
 func (syncer *DefaultSyncer) HandleNewTipset(ctx context.Context, tipsetCids types.SortedCidSet) error {
-	logSyncer.Debugf("trying to sync %v\n", tipsetCids)
+	logSyncer.Debugf("trying to sync %v", tipsetCids)
```

Comment:
It could be helpful to make this more precise:
"Beginning fetch and sync of chain with head %v"